products/pdfmorph.rst

```diff
@@ -0,0 +1,142 @@
+.. include:: ../abbreviations.txt
+
+########
+PDFmorph
+########
+
+Overview
+========
+
+.. note::
+
+	PDFmorph is currently run from the command line, which requires
+	opening and typing into a terminal window or Windows command
+	prompt. It is recommended that you consult online resources and
+	become somewhat familiar before using PDFmorph.
+
+PDFmorph is a Python software package designed to increase the insight
+researchers can obtain from measured atomic pair distribution functions
+(PDFs) in a model-independent way. The program was designed to help a
+researcher answer the question: "Has my material undergone a phase
+transition between these two measurements?"
+
+PDFmorph makes use of several data manipulation techniques to correct
+for benign effects such as thermal expansion (peak shift) and increased
+thermal motion (peak broadening) or a cahnge in scale due to differences
+in incident flux, for example. PDFmorph will do its best to correct
+for these benign effects before computing and plotting a difference
+curve.
+
+One PDF (typically that collected at higher temperature) is identified
+as the "target" PDF and the second is "morphed" by "stretching" (changing
+the r-axis to simulate a uniform lattice expansion), "smearing"
+(broadening peaks through a uniform convolution to simulate increased
+thermal motion), and "scaling" (self-explanatory). PDFmorph will vary
+amplitude of morphing transformations to obtain the best fit between
+morphed and target PDFs, then plot them along with the difference curve
+plotted below.
+
+There are also a few other morphing transformations in the program.
+
+Finally, we note that PDFmorph should work on other spectra that are
+not PDFs, though it has not been extensively tested beyond the PDF.
+
+.. PDFmorph will soon be included in :doc:`xPDFsuite <xPDFsuite>`, a powerful end-to-end software for X-ray PDF analysis. xPDFsuite is designed for flexible, high throughput PDF analyses and contains many features for interrogating, comparing, and modeling data as well as simply doing data reduction.
+
+.. figure:: ../images/pdfmorph_nacl_ex.png
+	:align: center
+
+	Transformation techniques (scale, smear, and stretch, respectively)
+	utilized by PDFmorph applied sequentially on an NaCl sample
+
+
+Installation
+============
+
+By downloading and using this software, you are agreeing to the conditions
+specified in the :doc:`software license <pdfmorph_license>`.
+
+In order to take automatically take care of software dependency
+downloads, we recomment you install PDFmorph using conda. To do that,
+you will first have to download and install *Anaconda* or *mini-conda*
+from `continuum <https://www.continuum.io>`__.
+
+PDFmorph is available for all operating systems supported by Anaconda,
```

Comment:
remove "supported by Anaconda".  It is not an anaconda issue.  Either just say "PDFmorph is available on 32 and 64-bit Linux, Mac OS X, and 32 and 64-bit Windows" or "all operating systems that python runs on".  But if there is some dependency that it doesn't run on this is incorrect, so strictly we should list the OS's that we test builds on using Travis....as we can only guarantee that it runs on those systems.

Reply:
I'd suggest to not mention any operating systems at all.
PDFmorph and its dependencies are pure Python (diffpy.utils) or very common (matplotlib, numpy) so there is not really as on what OS this can run.

Reply:
Took them out overall!